include/dlaf/matrix/distribution.h

```diff
@@ -148,504 +226,786 @@ class Distribution {
-  GlobalTileSize global_nr_tiles_;
+  GlobalTileSize nr_tiles_;
   LocalTileSize local_nr_tiles_;
   TileElementSize block_size_;
   TileElementSize tile_size_;
 
   comm::Index2D rank_index_;
   comm::Size2D grid_size_;
   comm::Index2D source_rank_index_;
+
+public:
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const LocalElementSize& localSize() const noexcept {
+    return local_size();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const GlobalTileSize& nrTiles() const noexcept {
+    return nr_tiles();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const LocalTileSize& localNrTiles() const noexcept {
+    return local_nr_tiles();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const TileElementSize& blockSize() const noexcept {
+    return block_size();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("use tile_size method")
+  const TileElementSize& baseTileSize() const noexcept {
+    return tile_size();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const comm::Index2D& rankIndex() const noexcept {
+    return rank_index();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("use grid_size method")
+  const comm::Size2D& commGridSize() const noexcept {
+    return grid_size();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  const comm::Index2D& sourceRankIndex() const noexcept {
+    return source_rank_index();
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  GlobalElementIndex globalElementIndex(const GlobalTileIndex& global_tile,
+                                        const TileElementIndex& tile_element) const noexcept {
+    return global_element_index(global_tile, tile_element);
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  GlobalTileIndex globalTileIndex(const GlobalElementIndex& global_element) const noexcept {
+    return global_tile_index(global_element);
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  GlobalTileIndex globalTileIndex(const LocalTileIndex& local_tile) const noexcept {
+    return global_tile_index(local_tile);
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  comm::Index2D rankGlobalTile(const GlobalTileIndex& global_tile) const noexcept {
+    return rank_global_tile(global_tile);
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  LocalTileIndex localTileIndex(const GlobalTileIndex& global_tile) const {
+    return local_tile_index(global_tile);
+  }
+
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  TileElementIndex tileElementIndex(const GlobalElementIndex& global_element) const noexcept {
+    return tile_element_index(global_element);
+  }
+
+  TileElementSize tileSize(const GlobalTileIndex& index) const noexcept {
```

Comment:
Why isn't this going to be deprecated? Shouldn't we use `tile_size_of` in the future?